STUDIO/org.openl.rules.webstudio/src/org/openl/rules/rest/UsersService.java

```diff
@@ -303,29 +313,65 @@ public void deleteUser(@PathParam("username") String username) {
         return options;
     }
 
+    @GET
+    @Path("/{username}/groups/external")
+    public Set<String> getUserGroupsGroups(@PathParam("username") String username,
+            @QueryParam("matched") Boolean matched) {
+        SecurityChecker.allow(Privileges.ADMIN);
+        checkUserExists(username);
+        List<Group> extGroups;
+        if (matched == null) {
+            extGroups = extGroupService.findAllForUser(username);
+        } else if (Boolean.FALSE.equals(matched)) {
```

Comment:
You already checked above that matched isn't null,
So I think you can simplify expression to this:
`} else if (!matched) {`